src/main/resources/assets/theoneprobe/lang/de_de.json

```diff
@@ -8,7 +8,7 @@
     "item.theoneprobe.gold_helmet_probe": "Goldhelm mit Sonde",
     "item.theoneprobe.iron_helmet_probe": "Eisenhelm mit Sonde",
     "item.theoneprobe.creativeprobe": "Die Kreativsonde",
-    "item.theoneprobe.probenote": "The One Probe-Liesmich",
+    "item.theoneprobe.probenote": "Handbuch: Die einen Sonde",
```

Comment:
I feel like it should be: "Handbuch: The One Probe" since "The One Probe" is a proper noun.